### PR TITLE
[v6r17] Adding version switch to install_site.sh

### PR DESCRIPTION
**Pull request by fstagni**
For issue https://github.com/DIRACGrid/DIRAC/issues/3171

Also mod documentation and Jenkins invocation.

### CODE REVIEW

**Comment by fstagni** [comment]
Review (approve!) anyone?

File: tests/Jenkins/dirac_ci.sh [hunk 1]
```diff
@@ -113,7 +113,7 @@ function installSite(){
   sed -i s/VAR_NoSQLDB_Port/$NoSQLDB_PORT/g $SERVERINSTALLDIR/install.cfg
 
   echo '==> Started installing'
-  $SERVERINSTALLDIR/install_site.sh $SERVERINSTALLDIR/install.cfg
+  $SERVERINSTALLDIR/install_site.sh -v $projectVersion $SERVERINSTALLDIR/install.cfg
```
Comment (chaen) [inline tests/Jenkins/dirac_ci.sh]: Where do you define this variable ? In the jenkins configuration ?

Comment (fstagni) [inline tests/Jenkins/dirac_ci.sh]: This is defined by function "findRelease" (which is in file utilities.sh

**Comment by chaen** [comment]
Review OK

**Pull request merged**